File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DIEUCHUYENService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DIEUCHUYENService : BaseService<NS_DIEUCHUYEN, INS_DIEUCHUYENRepository>
    , INS_DIEUCHUYENService
    {
    private readonly INS_DIEUCHUYENRepository _NS_DIEUCHUYENRepository;
    public NS_DIEUCHUYENService(INS_DIEUCHUYENRepository NS_DIEUCHUYENRepository) : base(NS_DIEUCHUYENRepository)
    {
    _NS_DIEUCHUYENRepository = NS_DIEUCHUYENRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DIEUCHUYEN.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DIEUCHUYEN } from './NS_DIEUCHUYEN.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DIEUCHUYENService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'MaNhanVien', 'NgayDieuChuyen', 'Ngayhieuluc', 'MaPhongBanCu', 'MaChucVuCu', 'MaPhongBanMoi', 'MaChucVuMoi'];

    List: NS_DIEUCHUYEN[] | undefined;
    ListFilter: NS_DIEUCHUYEN[] | undefined;
    FormData!: NS_DIEUCHUYEN;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DIEUCHUYEN";
    }
}

